### ISSUE TITLE
URL in address bar cannot be copied with Cmd+C using Bluetooth Keyboard

### ISSUE THREAD

**Issue by sirn** [mozilla-mobile/firefox-ios#5071]
**Steps to reproduce:**

1. Open any web page
2. Focus address bar (e.g. Cmd+L)
3. Press Cmd+C to copy the URL

**Expected behavior:**

URL is copied to clipboard.

**Actual behavior:**

Nothing is copied.

**Comment by sirn** [comment]
After investigating a bit more, it seems like the only way I can reliably copy an address is:

1. Open a webpage
2. Press Cmd+L to focus address bar
3. Press arrow key to _unselect the text_ 
4. Press Cmd+A to select all text
5. Press Cmd+C to copy the URL
